Network/Public/Include/Network/iAPI.hpp

```diff
@@ -0,0 +1,184 @@
+            void messageAllClients              (std::shared_ptr<Network::Connection> exceptionConnection,const Message& message) const override;   \
+            void checkServerPing                (std::shared_ptr<Network::Connection> clientConnection,   const Message& message) const override;   \
+            void readAllMessage                 (std::shared_ptr<Network::Connection> clientConnection,   const Message& message) const override;   \
+            void channelListRequest             (std::shared_ptr<Network::Connection> clientConnection) const override;                             \
+            void messageHistoryRequest          (std::shared_ptr<Network::Connection> clientConnection,   const Message& message) const override;   \
+            void messageStoreRequest            (std::shared_ptr<Network::Connection> clientConnection,   const Message& message) const override;   \
+            void replyHistoryRequest            (std::shared_ptr<Network::Connection> clientConnection,   const Message& message) const override;   \
+            void replyStoreRequest              (std::shared_ptr<Network::Connection> clientConnection,   const Message& message) const override;   \
+            void messageDeleteRequest           (std::shared_ptr<Network::Connection> clientConnection,   const Message& message) const override;   \
+            void messageEditRequest             (std::shared_ptr<Network::Connection> clientConnection,   const Message& message) const override;   \
+            void messageReactionRequest         (std::shared_ptr<Network::Connection> clientConnection,   const Message& message) const override;   \
+            void registrationRequest            (std::shared_ptr<Network::Connection> clientConnection,   const Message& message) const override;   \
+            void loginRequest                   (std::shared_ptr<Network::Connection> clientConnection,   const Message& message) const override;   \
+            void channelLeaveRequest            (std::shared_ptr<Network::Connection> clientConnection,   const Message& message) const override;   \
+            void channelSubscribeRequest        (std::shared_ptr<Network::Connection> clientConnection,   const Message& message) const override;   \
+            void channelSubscriptionListRequest (std::shared_ptr<Network::Connection> clientConnection) const override;                             \
+            void channelDeleteRequest           (std::shared_ptr<Network::Connection> clientConnection,   const Message& message) const override;   \
+            void channelCreateRequest           (std::shared_ptr<Network::Connection> clientConnection,   const Message& message) const override;   \
+            void directMessageCreateRequest     (std::shared_ptr<Network::Connection> clientConnection,   const Message& message) const override;   \
+            void defaultRequest                 () const override;
+
+
+
```

Comment:
remove useless spaces

Reply:
No. This was added for easy reading

Reply:
![image](https://user-images.githubusercontent.com/70116007/209483290-4bc3df71-3227-4a37-9af4-99f6f05313d6.png)
If you're already doing it this way, you should move "const ovveride" to the rest, like the form.